staging/spago.dhall

```diff
@@ -0,0 +1,52 @@
+{ name = "try-purescript-server"
+, dependencies =
+    [ "arrays"
+    , "bifunctors"
+    , "console"
+    , "const"
+    , "contravariant"
+    , "control"
+    , "distributive"
+    , "effect"
+    , "either"
+    , "enums"
+    , "exceptions"
+    , "exists"
+    , "foldable-traversable"
+    , "foreign"
+    , "foreign-object"
+    , "free"
+    , "functions"
+    , "functors"
+    , "generics-rep"
+    , "globals"
+    , "identity"
+    , "integers"
+    , "lazy"
+    , "math"
+    , "maybe"
+    , "ordered-collections"
+    , "prelude"
+    , "profunctor"
+    , "proxy"
+    , "quickcheck"
+    , "random"
+    , "react"
+    , "react-dom"
```

Comment:
Is this mainly for testing the shim stuff? Do we intend to have a process for adding additional libraries from the set. Or perhaps we should just blanket state we are not adding more libraries until we figure out a caching/purging solution for externs?

Reply:
I think I originally added these a week or two ago for testing the shim stuff, yeah. I'd like to at least add back libraries which were previously provided by other backends (#142). After that, saying we are not going to add any more libraries until we figure out a caching/purging solution (at which point we can add the whole package set) works for me.